godtools/Localizable.strings

```diff
@@ -0,0 +1,24 @@
+/* 
+  Localizable.strings
```

Comment:
So in the xibs and the code, I think you could still do `"language_settigs_title"`.localized and I think that would be ok.